tests/conftest.py

```diff
@@ -7,10 +7,39 @@
 from time import sleep
 
 import pytest
+from aiosmtpd.controller import Controller
 
 from pocketbase import PocketBase
 from tests.prep import ensure_pocketbase_executable
 
+SMTP_PORT = 8025
```

Comment:
I'd like you to use a "pick free port" mechanism for this, the function is already included in this file.

Reply:
See updated PR